Source/Models.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  Models.swift
```

Comment:
Sure, if you don't mind. Primarily just to make it easier to nail down what happens where if someone wants to debug something or contribute. Thanks :)